### PR TITLE
Send screenshots from Taskcluster to wpt.fyi

### PR DESCRIPTION
**Pull request by Hexcles**
Part of web-platform-tests/wpt.fyi#57

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -84,6 +84,7 @@ tasks:
                     --
                     --channel=${browser.channel}
                     --log-wptreport=../artifacts/wpt_report.json
+                    --log-wptscreenshot=../artifacts/wpt_screenshot.db
```
Comment (jgraham) [inline .taskcluster.yml]: I don't really like the `.db` extension, but I guess you've hardcoded this elsewhere now so meh.

**Review by jgraham — APPROVED** [review]

File: .taskcluster.yml (hunk 1 above)
Comment (Hexcles) [reply .taskcluster.yml]: @jgraham not really. Any suggestions?

Comment (gsnedders) [reply .taskcluster.yml]: From IRC:

15:03 < jgraham> Hexcles: Honestly if it's a made-up format I'd be pretty happy with no extension
15:07 < Hexcles> jgraham: hmm what about txt?
15:09 < jgraham> Hexcles: txt is fine […]

**Comment by Hexcles** [comment]
Force-merged because GitHub APIs (and therefore, CIs) were totally busted.

**Pull request merged**